ledger/catchpointtracker_test.go

```diff
@@ -2094,3 +2097,47 @@ func TestMakeCatchpointFilePath(t *testing.T) {
 	}
 
 }
+
+// test a case where in-memory SQLite, combined with fast locking (no deadlock detection)
+func TestCatchpointTrackerFastRoundsDBRetry(t *testing.T) {
+	partitiontest.PartitionTest(t)
+
+	var bufNewLogger bytes.Buffer
+	log := logging.NewLogger()
+	log.SetOutput(&bufNewLogger)
+
+	deadlock.Opts.Disable = true // disable deadlock detection during this test
```

Comment:
This is not a parallel test so we shouldn't have to consider deadlock detection being disabled elsewhere.